server/gokbg3/grails-app/services/org/gokb/PackageService.groovy

```diff
@@ -1693,7 +1673,7 @@ class PackageService {
     StringBuilder name = new StringBuilder()
     if (type == ExportType.KBART) {
       name.append(toCamelCase(pkg.provider?.name ? pkg.provider.name : "unknown Provider")).append('_')
-          .append(toCamelCase(pkg.scope.value)).append('_')
+          .append(toCamelCase(pkg.scope?.value ?: '')).append('_')
```

Comment:
check presence of scope : pkg.scope ? pkg.scope.value : '', counts also for line above